extensionsGallery-insider.json

```diff
@@ -3206,8 +3206,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.9",
-							"lastUpdated": "11/29/2021",
+							"version": "0.2.10",
```

Comment:
The URL still seems to be pointing to the 0.2.9 version of the VSIX, is that going to be updated once this is merged in?

Reply:
The URL redirect sometimes caches, if you open the link in a different browser it should now download the latest version.